tests/test_coriolis.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pytest
+from pytest import param as p
+
+from jaxsim.high_level.common import VelRepr
+from jaxsim.high_level.model import Model
+
+from . import utils_models, utils_rng
+from .utils_models import Robot
+
+
+@pytest.mark.parametrize(
+    "robot, vel_repr",
+    [
+        p(*[Robot.DoublePendulum, VelRepr.Inertial], id="DoublePendulum-Inertial"),
+        p(*[Robot.DoublePendulum, VelRepr.Body], id="DoublePendulum-Body"),
+        p(*[Robot.DoublePendulum, VelRepr.Mixed], id="DoublePendulum-Mixed"),
+        p(*[Robot.Ur10, VelRepr.Inertial], id="Ur10-Inertial"),
+        p(*[Robot.Ur10, VelRepr.Body], id="Ur10-Body"),
+        p(*[Robot.Ur10, VelRepr.Mixed], id="Ur10-Mixed"),
+        p(*[Robot.AnymalC, VelRepr.Inertial], id="AnymalC-Inertial"),
+        p(*[Robot.AnymalC, VelRepr.Body], id="AnymalC-Body"),
+        p(*[Robot.AnymalC, VelRepr.Mixed], id="AnymalC-Mixed"),
+        p(*[Robot.Cassie, VelRepr.Inertial], id="Cassie-Inertial"),
+        p(*[Robot.Cassie, VelRepr.Body], id="Cassie-Body"),
+        p(*[Robot.Cassie, VelRepr.Mixed], id="Cassie-Mixed"),
```

Comment:
Perhaps we can test against also the iCub/ergoCub models, as most probably those are the one we are going to use the feature against?

Kind of unrelated to the PR, but just because I realized it: testing real world models is ok, but in my experience you may have some blinds spots. In iDynTree I historically found many problems with programmatically generated random models, generated by [`getRandomModel`](https://github.com/robotology/idyntree/blob/4a981e11e282ab5c10009523157ba151b2519509/src/model/include/iDynTree/ModelTestUtils.h#L123) function. 

It could make sense to do something similar here, or also directly use it here as we can export the randomly generated model to URDF, something like
~~~python
import idyntree.bindings as idt
nrOfJoints = 20
randomModel = idt.getRandomModel(nrOfJoints)
mdlExport = idt.ModelExporter()
mdlExport.init(randomModel)
urdfString = "dummy"
urdfFilePath = "/dummy/dummy"
mdlExport.exportModelToString(urdfString)
# or
mdlExport.exportModelToFile(urdfFilePath)
~~~

Reply:
Thanks Silvio for your suggestion, I totally agree with this! I will open an issue to address it

Reply:
Issue opened https://github.com/ami-iit/jaxsim/issues/76